ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
-1 I don't think it helps to remove them unless you're going to replace them with similar functionality.